layouts/partials/header-menu.html

```diff
@@ -1,4 +1,27 @@
 <ul>
-    <li><a href="{{ "about" | relURL }}">{{ i18n "about" }}</a></li>
-    <li><a href="{{ "/" | relURL }}">{{ i18n "posts" }}</a></li>
+    {{ $currentPage := . }}
+    {{ range .Site.Menus.main }}
+        {{ if .HasChildren }}
+            <li class="{{ if $currentPage.HasMenuCurrent "main" . }}active{{ end }}">
+                <a href="#">
+                    {{ .Pre }}
+                    <span>{{ .Name }}</span>
+                </a>
+            </li>
+            <ul class="sub-menu">
+                {{ range .Children }}
+                    <li class="{{ if $currentPage.IsMenuCurrent "main" . }}active{{ end }}">
+                        <a href="{{ .URL }}">{{ .Name }}</a>
+                    </li>
+                {{ end }}
+            </ul>
```

Comment:
Creating submenus looks a bit weird. For example, add the following to your config file:

```
[menu]
  [[menu.main]]
    identifier = "about"
    name = "About"
    title = "About"
    url = "/about/"
    weight = 0
  [[menu.main]]
    identifier = "posts"
    name = "Posts"
    title = "Posts"
    url = "/posts/"
    weight = 0
  [[menu.main]]
    identifier = "topic"
    parent = "posts"
    name = "Topic"
    title = "Topic"
    url = "/posts/topic/"
    weight = 0
  [[menu.main]]
    identifier = "anothertopic"
    parent = "posts"
    name = "Another topic"
    title = "Another topic"
    url = "/posts/anothertopic/"
    weight = 0
```

I don't think the theme is very well suited for nested menu structures at the moment, so I would suggest to remove this part.

The CSS class `active` is also not used in Tale, but removing this entire section already resolves that issue.

Reply:
Removed submenu.